src/custom/components/CurrencyInputPanel/index.tsx

```diff
@@ -88,8 +126,10 @@ export const Wrapper = styled.div<{ selected: boolean }>`
 
 export function CurrencyInputPanel(props: CurrencyInputPanelProps) {
   const { currency } = props
+  const isRefreshingQuote = useIsQuoteRefreshing()
+  const showLoader = useLoadingWithTimeout(isRefreshingQuote, 4000)
```

Comment:
The `4000` is the same number as we use in https://github.com/gnosis/cowswap/pull/757/files#diff-ccd4342ee368f98ada541211f768c399ab70fa6b0b405f8401fb6449f28a1abaR12

Should we use a global const/theme var to have a consistent number?